pkg/ee/metering/reconcile.go

```diff
@@ -28,46 +28,41 @@ import (
 	"context"
 	"fmt"
 
-	"go.uber.org/zap"
-
 	kubermaticv1 "k8c.io/kubermatic/v2/pkg/crd/kubermatic/v1"
-	operatorv1alpha1 "k8c.io/kubermatic/v2/pkg/crd/operator/v1alpha1"
 	"k8c.io/kubermatic/v2/pkg/resources/reconciling"
 
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	ctrlruntimeclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 // ReconcileMeteringResources reconciles the metering related resources.
-func ReconcileMeteringResources(ctx context.Context, client ctrlruntimeclient.Client, namespace string,
-	cfg *operatorv1alpha1.KubermaticConfiguration, seed *kubermaticv1.Seed, log *zap.SugaredLogger) error {
+func ReconcileMeteringResources(ctx context.Context, client ctrlruntimeclient.Client, seed *kubermaticv1.Seed) error {
 
 	if err := persistentVolumeClaimCreator(ctx, client, seed); err != nil {
-		return fmt.Errorf("failed to reconcile metering pvc: %v", err)
+		return fmt.Errorf("failed to reconcile metering PVC: %v", err)
 	}
 
 	if err := reconciling.ReconcileServiceAccounts(ctx, []reconciling.NamedServiceAccountCreatorGetter{
 		serviceAccountCreator(),
-	}, metav1.NamespaceSystem, client); err != nil {
+	}, seed.Namespace, client); err != nil {
 		return fmt.Errorf("failed to reconcile metering ServiceAccounts: %v", err)
 	}
 
 	if err := reconciling.ReconcileClusterRoleBindings(ctx, []reconciling.NamedClusterRoleBindingCreatorGetter{
-		clusterRoleBindingCreator(cfg.Namespace),
+		clusterRoleBindingCreator(seed.Namespace),
```

Comment:
Why you are not using the k8c.io/kubermatic/v2/pkg/resources.KubermaticNamespace ?